vuepress/docs/next/tutorials/solution/pda-tutorial.md

```diff
@@ -36,7 +36,7 @@ There are numerous assets installed as part of the Entando PDA plugin. Entando B
      2. In the pop-up window, enter `Entando Hub` and [https://hub.entando.com] for the URL, then `Save` 
      3. Click on the Hub in the Registry 
 
-2. From the Hub Catalog, `Deploy` and `Install` the PDA bundle. It will take a few minutes to install.
+2. From the Hub Catalog, `Deploy` and `Install` the PDA bundle. The installation may take several minutes while the application downloads the Linux images for the microservices and installs the related assets.
```

Comment:
Just one microservice in this case so singular -  "Linux image for the microservice"